explorer/src/Stories/Status.elm

```diff
@@ -29,4 +30,8 @@ stories =
           , \_ -> Status.blue "Status" []
           , {}
           )
+        , ( "Mixed"
+          , \_ -> Status.twoColors "Control prerequisites" Color.yellowStatus (Color.yellow |> Color.withAlpha 0.4) 60.0 []
```

Comment:
Could we use the `progress` html tag for this case? Then I think screen readers can pick up the semantic meaning

Reply:
wasn't aware of that tag, nice!

Reply:
I see this tag is problematic in some ways. Firefox does not have support for pseudo elements like "before" and I see using custom html/css for this is recommended. 
https://stackoverflow.com/questions/41429906/how-to-display-data-label-inside-html5-progress-bar-cross-browser/41430146#41430146
https://stackoverflow.com/questions/53278520/how-do-i-put-text-into-a-progress-bar-in-html5-with-bulma

Reply:
It is good on chrome and can be worked out 
![image](https://github.com/SGFinansAS/elm-components/assets/34012758/967f2ab2-14fa-4088-9397-78713cb7c551)

but on firefox it is pretty bad without the pseudo elements
![image](https://github.com/SGFinansAS/elm-components/assets/34012758/092b7ed7-0741-4998-ba0f-b689f25553ef)

Reply:
Ahh ok. Should make sure we have sufficient meta data for accessibility then. Maybe the data-label they use in that thread is enough. There is also a progressbar role: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/progressbar_role

Reply:
I see the concern around accessibility. I have made modifications and for example when I click on a row of this table, it reads the columns one by one and for this one it says "Kontroller forutsetninger 50 percent". I think it is good.
![image](https://github.com/SGFinansAS/elm-components/assets/34012758/1c393180-bf56-405b-a592-4f3e7d4abe75)

Reply:
Nice!